cmd/kusk/cmd/dashboard.go

```diff
@@ -171,3 +179,24 @@ func getBrowserOpenCmdAndArgs(url string) (string, []string) {
 
 	return cmd, args
 }
+
+func localPortCheck(port int) error {
+	ln, err := net.Listen("tcp", ":"+fmt.Sprint(port))
+	if err != nil {
+		fmt.Println(err)
```

Comment:
You really want to keep printing the `error` if a port has been allocated?

Reply:
no :D